systest/build.rs

```diff
@@ -60,6 +60,18 @@ fn main() {
     });
 
     cfg.skip_const(move |s| {
+        if version < 64 {
```

Comment:
I doubt the version here is exactly when these consts were introduced. I'm hoping that doesn't matter.